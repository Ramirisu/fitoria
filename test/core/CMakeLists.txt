fitoria_add_test(NAME test_dynamic_buffer SRCS test_dynamic_buffer.cpp)
fitoria_add_test(NAME test_fixed_string SRCS test_fixed_string.cpp)
fitoria_add_test(NAME test_from_string SRCS test_from_string.cpp)
fitoria_add_test(NAME test_net_io_context SRCS test_net_io_context.cpp)
fitoria_add_test(NAME test_net SRCS test_net.cpp)
fitoria_add_test(NAME test_strings SRCS test_strings.cpp)
fitoria_add_test(NAME test_type_traits SRCS test_type_traits.cpp)

add_subdirectory(expected)
add_subdirectory(optional)
