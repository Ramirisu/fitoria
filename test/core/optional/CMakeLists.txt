fitoria_add_test(NAME test_optional SRCS test_optional.cpp)
fitoria_add_test(NAME test_optional_void SRCS test_optional_void.cpp)
fitoria_add_test(NAME test_optional_lref SRCS test_optional_lref.cpp)
