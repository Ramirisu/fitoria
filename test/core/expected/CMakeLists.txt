fitoria_add_test(NAME test_expected_lref SRCS test_expected_lref.cpp)
fitoria_add_test(NAME test_expected_void SRCS test_expected_void.cpp)
fitoria_add_test(NAME test_expected SRCS test_expected.cpp)
